renovate.json

```diff
@@ -1,5 +1,22 @@
 {
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "packagePatterns": ["eslint"],
+      "groupName": "eslint",
+      "automerge": true,
+      "automergeType": "branch"
+    },
+    {
+      "packageNames": [
+        "delay",
+        "ava",
+        "p-defer"
+      ],
+      "automerge": true,
+      "automergeType": "branch"
+    }
```

Comment:
what about `husky` and `lint-staged`?

Reply:
those are not tested by the ci. should ideally be tested locally. or at least reviewed by hand

Reply:
alright, thanks for clarifying!